NBitcoin/BIP174/PartiallySignedTransaction.cs

```diff
@@ -1493,25 +1512,36 @@ public PSBT CoinJoin(PSBT other)
 			return result;
 		}
 
-		public PSBT TryFinalize(out bool result)
+		/// <summary>
+		/// If this method throws an error, that is a bug.
+		/// </summary>
+		/// <param name="errors"></param>
+		/// <returns></returns>
+		public PSBT TryFinalize(out InvalidOperationException[] errors)
```

Comment:
TryFoo() methods in C# usually work the other way around: they return bool, and have an out parameter for the thing to return

Reply:
Good point

Reply:
Will create another PR for this.